tests/e2e-instrumentation/instrumentation-apache-httpd/01-assert.yaml

```diff
@@ -83,12 +79,29 @@ status:
   - name: myapp
     ready: true
     started: true
-  - name: otc-container
-    ready: true
-    started: true
   initContainerStatuses:
   - name: otel-agent-source-container-clone
     ready: true
   - name: otel-agent-attach-apache
     ready: true
   phase: Running
+---
+apiVersion: apps/v1
```

Comment:
For deployment and service we can assert the status and ports as well.
examples:
https://github.com/open-telemetry/opentelemetry-operator/blob/main/tests/e2e-openshift/monitoring/01-assert.yaml#L10
https://github.com/open-telemetry/opentelemetry-operator/blob/main/tests/e2e-openshift/monitoring/01-assert.yaml#L45

Reply:
Thanks @IshwarKanse, I extended the asserts: https://github.com/open-telemetry/opentelemetry-operator/pull/4310/commits/d1faf5b65fff08fd194fb80e0d8be5c7f3e2983b